dev/docker/docker-compose.dev.yml

```diff
@@ -1,5 +1,32 @@
 version: "3"
 services:
+  keycloak:
+    image: quay.io/keycloak/keycloak:23.0.5
```

Comment:
I would maybe suggest to create our own keycloak-derived docker image for this container as we did for the `openslides-proxy`. Name proposals:
  - `

Currently when thinking about running this in production, I (for example) do not like passing admin credentials plain text via env vars.\
The password should go into a secret. Then we can - in line with the other openslides services - set a var like `KEYCLOAK_ADMIN_PASSWORD_FILE=/run/secrets/.../keycloak_admin_password`.\
I do not see a mechanism like this in the official keycloak image on quay.io. We could implement this in our wrapper

Another benefit is we would have a good way of controlling the keycloak version being used without having to maintain many different compose files or k8s deployment definitions for it.

Reply:
I think this setup is great for development. No images build is required when changing the configuration script and it's easy to test with other keycloak versions.

Is the production environment directly derived from this? Perhaps we can build something different for production...